### PR TITLE
Add introduction and conclusion sections

### PR DESCRIPTION
**Pull request by MichalisPanayides**
- [x] Introduction
   - [x] General intro
   - [x] Background info using news reports
   - [x] Literature review: Game theory & Queueing theory
   - [x] Literature review: Multi-network queues
   - [x] Establish the research problem and specify objectives
   - [x] Map of the paper
- [x] Conclusion
   - [x] Paper overview
   - [x] Objectives
   - [x] Limitations
   - [x] Future work

### CODE REVIEW

File: sections/introduction/main.tex [hunk 1]
```diff
@@ -1 +1,97 @@
-\section{Introduction}
\ No newline at end of file
+\section{Introduction}
+
+A huge issue for ambulances in the United Kingdom is that they stay
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
An important issue for ambulances in the United Kingdom is that they stay 
```

File: sections/introduction/main.tex [hunk 2]
```diff
@@ -1 +1,97 @@
-\section{Introduction}
\ No newline at end of file
+\section{Introduction}
+
+A huge issue for ambulances in the United Kingdom is that they stay 
+parked outside hospitals for a huge amount of time.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: I feel a reviewer might ask for some sort of evidence of this. What would be great would be a reference in a journal where they analysed waiting of parked ambulances. If we cannot find that, let us put in one of the many news articles.

File: sections/introduction/main.tex [hunk 3]
```diff
@@ -1 +1,97 @@
...
+parked outside hospitals for a huge amount of time. 
+This is a problem for the ambulance service, since the longer ambulances are
+blocked in the hospital's parking space the longer the patients in them will
+have to wait to receive their treatment but also the longer future patients 
+will be stuck waiting for an ambulance.
+This paper aims to describe a game theoretic model informed by an underlying 
+queueing model motivated by this scenario.
+
+Emergency departments (EDs) are constantly under a lot of pressure to meet 
+targets and satisfy regulations \cite{EmergencyDepartmentWinterPressures}.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
targets and satisfy regulations~\cite{EmergencyDepartmentWinterPressures}.
```

This creates a space that won't be broken at a line. So you will never get a new line that just started with a reference.

File: sections/introduction/main.tex [hunk 4]
```diff
@@ -1 +1,97 @@
...
+There are various news reports that indicate that this is in fact an ongoing 
+issue that affects both the patients and the emergency medical services (EMS).
+Due to this blockage patients may spend a considerable amount of time in the 
+back of an ambulance waiting to be dispatched to the ED.
+Examples of such reports are \cite{mirror}, \cite{thenews} and \cite{bmj} where
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
Examples of such reports are \cite{mirror, thenews, bmj} where
```

File: sections/introduction/main.tex [hunk 5]
```diff
@@ -1 +1,97 @@
...
+the impact that this issue has on patients is shown. 
+Furthermore, this is also a huge issue for the EMS.
+By blocking the ambulances at the hospital's parking space, valuable time is 
+wasted where ambulances could be attending new patients \cite{eastanglia}.
+
+In the past a number of papers have been published that touch upon the use of
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
A number of papers have been published that touch upon the use of 
```

File: sections/introduction/main.tex [hunk 6]
```diff
@@ -1 +1,97 @@
...
+back of an ambulance waiting to be dispatched to the ED.
+Examples of such reports are \cite{mirror}, \cite{thenews} and \cite{bmj} where
+the impact that this issue has on patients is shown. 
+Furthermore, this is also a huge issue for the EMS.
+By blocking the ambulances at the hospital's parking space, valuable time is 
+wasted where ambulances could be attending new patients \cite{eastanglia}.
+
+In the past a number of papers have been published that touch upon the use of 
+queueing models together with game theoretic concepts.
+In \cite{FirmCompetition} the authors study a simultaneous price competition
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: This reference seems out of place as is. In the sentence above you mention how a number of papers are at the intersection of queueing models and game theory. Is this one of them?

File: sections/introduction/main.tex [hunk 7]
```diff
@@ -1 +1,97 @@
...
+A main result from this paper was that equilibria, where both firms operate
+together, are not socially optimal while when both firms operate as a monopoly 
+they are.
+Another extension of \cite{FirmCompetition} was introduced in 
+\cite{FirmCompetitionExtension} where a long-run version of the competition was 
+considered that also had capacity as a decision variable.
+In \cite{knight2017measuring} a normal form game is built that is informed by a 
+two-dimensional Markov chain in order to model interactions between critical
+care units.
+An additional paper that focuses on competition is \cite{fan2009short} where
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: The link to queues is not clear here.

File: sections/introduction/main.tex [hunk 8]
```diff
@@ -1 +1,97 @@
...
+the authors created a competition between 2 sellers with products having 
+different implementation costs.
+In \cite{sadat2015can} a healthcare application was studied where patients 
+could choose between two hospitals, where a utility function is derived that is
+based on patients' perceived quality of life.
```
Comment (drvinceknight) [inline sections/introduction/main.tex:60]: You should add the [PoA](https://www.sciencedirect.com/science/article/pii/S0377221713003019).

There are also a fair few healthcare ones that I think you've omitted (the ambulance diversion one specifically).

File: sections/introduction/main.tex [hunk 9]
```diff
@@ -1 +1,97 @@
...
+
+Another specific part of this research is the construction of a queueing system
+with two tandem queues.
+In \cite{d2015pure} the authors explore threshold joining strategies in a 
+Markov model that has two tandem queues.
+Another great example is the one described in \cite{burnetas2013customer}
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: Tell me about this paper. (No need to respond, just one that I'd like us to discuss.)

File: sections/introduction/main.tex [hunk 10]
```diff
@@ -1 +1,97 @@
...
+where they investigated a network of multiple tandem queues where customers 
+decide which queue to attend before joining.
+Similarly, in \cite{bacsar2002stackelberg} the authors examine a network of
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: Same as above, are you sure it's tandem?

File: sections/introduction/main.tex [hunk 11]
```diff
@@ -1 +1,97 @@
...
+Another great example is the one described in \cite{burnetas2013customer}
+where they investigated a network of multiple tandem queues where customers 
+decide which queue to attend before joining.
+Similarly, in \cite{bacsar2002stackelberg} the authors examine a network of 
+\(N\) tandem M/M/1 queues and with multi-type customers. 
+The customers in this paper react to a price \(p\) by picking demand rates that 
+maximise utility.
+In \cite{veltman2005equilibrium} a profit maximisation problem is studied that
+has an M/M/1 queue and a parking service providing complementary service while
+the customer is in service.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: What do they do?

File: sections/introduction/main.tex [hunk 12]
```diff
@@ -1 +1,97 @@
...
+Similarly, in \cite{bacsar2002stackelberg} the authors examine a network of 
+\(N\) tandem M/M/1 queues and with multi-type customers. 
+The customers in this paper react to a price \(p\) by picking demand rates that 
+maximise utility.
+In \cite{veltman2005equilibrium} a profit maximisation problem is studied that
+has an M/M/1 queue and a parking service providing complementary service while
+the customer is in service. 
+The problem was later extended by \cite{sun2009equilibrium} where they 
+considered arrivals of batches that can share the parking service.
+Finally, \cite{afeche2007decentralized} examines a tandem network of two M/M/1
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: Same comment as above

File: sections/introduction/main.tex [hunk 13]
```diff
@@ -1 +1,97 @@
...
+care units.
+An additional paper that focuses on competition is \cite{fan2009short} where
+the authors created a competition between 2 sellers with products having 
+different implementation costs.
+In \cite{sadat2015can} a healthcare application was studied where patients 
+could choose between two hospitals, where a utility function is derived that is
+based on patients' perceived quality of life.
+
+Another specific part of this research is the construction of a queueing system
+with two tandem queues.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: Let us say here what we mean by tandem queues. Arguably your problem is not a tandem queue.

File: sections/introduction/main.tex [hunk 14]
```diff
@@ -1 +1,97 @@
...
+has an M/M/1 queue and a parking service providing complementary service while
+the customer is in service. 
+The problem was later extended by \cite{sun2009equilibrium} where they 
+considered arrivals of batches that can share the parking service.
+Finally, \cite{afeche2007decentralized} examines a tandem network of two M/M/1 
+queues that are ran by two different profit-maximising service providers and 
+receive three different types of customers.
+
+The EDs in the United Kingdom have to follow some set of regulations imposed to 
+them by the NHS.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
them by the National Health Service (NHS).
```

File: sections/introduction/main.tex [hunk 15]
```diff
@@ -1 +1,97 @@
...
+One of these regulations is that 95\% of patients that arrive at the ED should 
+be admitted, transferred or discharged within four hours.
+This is where gaming behaviour might be observed between the EDs and the EMS.
+An assumption of this work is that some managerial decision making is involved
+in choosing when to start blocking ambulances.
```
Comment (drvinceknight) [inline sections/introduction/main.tex:112]: `This is similar to~\cite{<that ambulance diversion paper>}...`

File: sections/introduction/main.tex [hunk 16]
```diff
@@ -1 +1,97 @@
...
+    \item The emergence of gaming behaviour between EDs and the EMS.
+    \item A queueing model with 2 consecutive waiting spaces where one would 
+    serve as a parking space for the ambulances
+\end{itemize}
+The focus of this research is the construction of a 3-player game theoretic 
+model between two queueing systems and a service that distributes individuals
+to them. 
+The resultant model will then be used to explore the emergent dynamics between 
+the three players.
+The model is then applied to the healthcare scenario between two EDs and the
```
Comment (drvinceknight) [inline sections/introduction/main.tex:132]: Highlight how this differs from what's in the literature.

File: sections/introduction/main.tex [hunk 17]
```diff
@@ -1 +1,97 @@
...
+The model is then applied to the healthcare scenario between two EDs and the 
+EMS while looking into the inefficiencies that emerge and ways to apply some 
+incentive mechanisms to improve them.
+
+This paper first gives an overview of the game theoretic model that was built, 
+then goes on to describe how the queueing models were constructed and then 
+follows with the methodology that was used to build the game.
+Finally, there is a section that describes how this game theoretic framework
+can be applied at the interface between the EMS and EDs together with some 
+analysis on the behaviour that is observed.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
This paper first gives an overview of the game theoretic model, 
then goes on to describe a novel queueing model.
Finally, there is a section that describes how this game theoretic framework
can be applied at the interface between the EMS and EDs together with some 
analysis on the behaviour that is observed.
```

File: sections/introduction/main.tex [hunk 18]
```diff
@@ -1 +1,97 @@
...
+    serve as a parking space for the ambulances
+\end{itemize}
+The focus of this research is the construction of a 3-player game theoretic 
+model between two queueing systems and a service that distributes individuals
+to them. 
+The resultant model will then be used to explore the emergent dynamics between 
+the three players.
+The model is then applied to the healthcare scenario between two EDs and the 
+EMS while looking into the inefficiencies that emerge and ways to apply some 
+incentive mechanisms to improve them.
```
Comment (drvinceknight) [inline sections/introduction/main.tex:134]: Be a bit louder here about the queueing model.

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: sections/introduction/main.tex (hunk 4 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: I did not know you can do this. Nice 👍

File: sections/introduction/main.tex (hunk 6 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Yes, this is one of them. Should I expand more on the queueing part?

File: sections/introduction/main.tex (hunk 7 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Similar to the previous comment ☝️. Should I expand on the queueing part of these?

By the way I got most of these from the book `Rational Queueing`

File: sections/introduction/main.tex (hunk 8 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex:60]: Will do

File: sections/introduction/main.tex (hunk 6 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: Yes I think so, as is what you have written is a description of any paper where there are two players in it.

File: sections/introduction/main.tex (hunk 7 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: Yup, just need to expand/clarify why the is relevant to queuing.

File: sections/introduction/main.tex (hunk 13 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Isn't it kind of a tandem queue though? Can I not label the model as a `network of tandem queues`?

File: sections/introduction/main.tex (hunk 9 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: `\cite{burnetas2013customer}` This one right?

Comment (drvinceknight) [reply sections/introduction/main.tex]: Yup.

File: sections/introduction/main.tex (hunk 17 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Should I not mention the methodology section at all here?

File: sections/introduction/main.tex (hunk 13 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: Yup it's absolutely a kind of tandem queue but it's really a tandem buffer with a **single** queue.

File: sections/introduction/main.tex (hunk 6 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Yep, I see what you are saying.

File: sections/introduction/main.tex (hunk 13 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Hmmmm. Should I just say that then?

```suggestion
with two tandem buffers.
```

File: sections/introduction/main.tex (hunk 17 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: No sorry, you should I removed that by mistake.

File: sections/introduction/main.tex (hunk 13 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: I don't know. I'm not actually sure what the best description of this is as to my knowledge it's the first time this has been done. Perhaps just what it is: `a tandem buffer with a single service centre`?

File: sections/introduction/main.tex (hunk 10 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: That's how the authors define it at the `problem formulation` section of that paper and that's also how `Hassin, R.` introduces it in `Rational Queueing`.

File: sections/introduction/main.tex (hunk 11 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Should I include more details on the methodology that has been used here?

File: sections/introduction/main.tex (hunk 12 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Same reply as above 😄

File: sections/introduction/main.tex (hunk 11 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: Just a bit but also perhaps the conclusions.

File: sections/introduction/main.tex (hunk 12 above)
Comment (drvinceknight) [reply sections/introduction/main.tex]: Same reply.

File: sections/introduction/main.tex (hunk 13 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: Sounds good. Will go with that for now then 👍

File: bibliography.bib [hunk 19]
```diff
@@ -309,6 +309,48 @@ @article{knight2017measuring
...
+  volume={57},
+  number={7},
+  pages={1300--1319},
+  year={2011},
+  publisher={INFORMS}
+}
+
 ###################
 ## NEWS ARTICLES ##
 ###################
```
Comment (drvinceknight) [inline bibliography.bib]: I believe `s/#/%` is more robust here.

File: sections/introduction/main.tex [hunk 20]
```diff
@@ -15,54 +17,82 @@ \section{Introduction}
...
-the authors created a competition between 2 sellers with products having 
-different implementation costs.
-In \cite{sadat2015can} a healthcare application was studied where patients 
+An additional paper that focuses on competition is~\cite{fan2009short} where
+the authors created a competition between two sellers where seller 1 supplies 
+a product instantly and seller 2 is modelled as a make-to-order M/M/1 queue.
+The game that is played requires the two sellers to make a choice on the price 
+of the product and then seller 2 to set a capacity that guarantees a maximum 
+expected delay.
+In~\cite{sadat2015can} a healthcare application was studied where patients
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: Something like:

```
In the above models, the players are attempting to increase 
their share of individuals choosing to queue.
In public healthcare type settings~\cite{knight2013selfish}, 
this is not necessarily the case. Players might not necessarily want and/or need 
to maximise so called `market share' and overall service needs to be considered as 
individuals aim to minimise experience congestion.
```

File: sections/introduction/main.tex [hunk 21]
```diff
@@ -15,54 +17,82 @@ \section{Introduction}
...
 could choose between two hospitals, where a utility function is derived that is
 based on patients' perceived quality of life.
+In~\cite{knight2013selfish} the impact that individual behaviour has on the 
+efficiency of public services is investigated.
+The authors place the individuals' choices between different public services 
+within the formulation of routing games and measure inefficiencies using a 
+concept known as the price of anarchy (\(PoA\))~\cite{koutsoupias1999worst}.
+They show that the price of anarchy increases with worth of service and that is
+low for systems with insufficient capacities.
+In \cite{deo2011centralized} the authors study the network effect of ambulance
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: `In~\cite{...`

File: sections/introduction/main.tex [hunk 22]
```diff
@@ -15,54 +17,82 @@ \section{Introduction}
...
+of the product and then seller 2 to set a capacity that guarantees a maximum 
+expected delay.
+In~\cite{sadat2015can} a healthcare application was studied where patients 
 could choose between two hospitals, where a utility function is derived that is
 based on patients' perceived quality of life.
+In~\cite{knight2013selfish} the impact that individual behaviour has on the 
+efficiency of public services is investigated.
+The authors place the individuals' choices between different public services 
+within the formulation of routing games and measure inefficiencies using a 
+concept known as the price of anarchy (\(PoA\))~\cite{koutsoupias1999worst}.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: `\(\text{PoA}\)` otherwise it looks like `P \times o \times A`.

File: sections/introduction/main.tex [hunk 23]
```diff
@@ -15,54 +17,82 @@ \section{Introduction}
...
+An additional paper that focuses on competition is~\cite{fan2009short} where
+the authors created a competition between two sellers where seller 1 supplies 
+a product instantly and seller 2 is modelled as a make-to-order M/M/1 queue.
+The game that is played requires the two sellers to make a choice on the price 
+of the product and then seller 2 to set a capacity that guarantees a maximum 
+expected delay.
+In~\cite{sadat2015can} a healthcare application was studied where patients 
 could choose between two hospitals, where a utility function is derived that is
 based on patients' perceived quality of life.
+In~\cite{knight2013selfish} the impact that individual behaviour has on the
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: `How individual behaviour based on congestion impacts the efficiency of public services is investigated in~\cite{knight2013selfish}.`

File: sections/introduction/main.tex [hunk 24]
```diff
@@ -15,54 +17,82 @@ \section{Introduction}
...
-Finally, \cite{afeche2007decentralized} examines a tandem network of two M/M/1 
-queues that are ran by two different profit-maximising service providers and 
-receive three different types of customers.
+Finally,~\cite{afeche2007decentralized} examines a tandem network of two M/M/1 
+queues that are ran by two different profit-maximising service providers.
+The network receives three types of customers; those requiring both services, 
+customers requiring the first service and customers requiring the second service.
+The authors showed that optimal prices also maximise social welfare and that
+removing two types of customers that don't need both services leads to higher 
+profit and lower demand rate.
```
Comment (drvinceknight) [inline sections/introduction/main.tex:104]:
```
The novelty of the queueing model here is a contribution to 
not only the game theoretic literature but also to the queueing theoretic literature.
To the authors knowledge, no such model of a tandem queueing model controlled 
by a strategic decision has been considered. % TODO Be very sure if this is correct 
```

File: sections/introduction/main.tex [hunk 25]
```diff
@@ -82,16 +113,18 @@ \section{Introduction}
 to them. 
 The resultant model will then be used to explore the emergent dynamics between 
 the three players.
+The new thing here is the use of a learning algorithm to get to the Nash 
+equilibrium.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```
While giving some consideration to equilibrium behaviour similarly to the 
work of~\cite{<a number of the paper you explored above>}`, emergent behaviour 
is more precisely addressed by considering a number of learning algorithms.
```

File: sections/introduction/main.tex [hunk 26]
```diff
@@ -82,16 +113,18 @@ \section{Introduction}
...
 The model is then applied to the healthcare scenario between two EDs and the 
 EMS while looking into the inefficiencies that emerge and ways to apply some 
 incentive mechanisms to improve them.
+The EDs are modelled as two queueing systems each with a tandem buffer and a 
+service centre where their performance measures of the queueing models are then
+used as the utilities of the game.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]: `service centre. The performance measures ...`

File: sections/introduction/main.tex [hunk 27]
```diff
@@ -82,16 +113,18 @@ \section{Introduction}
...
 
-This paper first gives an overview of the game theoretic model that was built, 
-then goes on to describe how the queueing models were constructed and then 
-follows with the methodology that was used to build the game.
+This paper first gives an overview of the game theoretic model, then goes on 
+to describe a novel queueing model, and then follows with the methodology that 
+was used to build the game.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```
and then follows with a number of theoretic results 
pertaining to the performance measures of the queuing model. 
These are used to build the utilities of the game.
```

**Review by drvinceknight — CHANGES_REQUESTED** [review]
I'm commenting on specific commits so cannot do this as suggestions.

File: bibliography.bib (hunk 19 above)
Comment (MichalisPanayides) [reply bibliography.bib]: What do you mean? Should I remove that comment? Or should I just say:

`# NEWS ARTICLES`

Comment (drvinceknight) [reply bibliography.bib]: No I believe technically you're using `#` for comments which could be brittle for the TeX compiler which would look for `%` as a comment.

Comment (MichalisPanayides) [reply bibliography.bib]: I think the .bib file doesn't support comments (might be wrong though). You can type anything outside of environment and wouldn't make a difference:

```
@article{key
   ...
}
```

But I guess I can use `%` for consistency

File: sections/introduction/main.tex (hunk 20 above)
Comment (MichalisPanayides) [reply sections/introduction/main.tex]: I will put this before the healthcare applications

File: sections/introduction/main.tex [hunk 28]
```diff
@@ -113,20 +122,30 @@ \section{Introduction}
...
 incentive mechanisms to improve them.
 The EDs are modelled as two queueing systems each with a tandem buffer and a 
-service centre where their performance measures of the queueing models are then
-used as the utilities of the game.
+service centre. 
+The performance measures are then used as the utilities of the game.
+The novelty of the queueing model here is a contribution to not only the game 
+theoretic literature but also to the queueing theoretic literature.
+To the authors knowledge, no such model of a tandem queueing model controlled 
+by a strategic decision has been considered.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
To the authors knowledge, no such model of a tandem queueing model with a pair of parameters for the buffer has been considered.
```

I think?

File: sections/introduction/main.tex [hunk 29]
```diff
@@ -48,23 +48,32 @@ \section{Introduction}
 The game that is played requires the two sellers to make a choice on the price 
 of the product and then seller 2 to set a capacity that guarantees a maximum 
 expected delay.
+In the above models, the players are attempting to increase their share of 
+individuals choosing to queue.
+In public healthcare type settings~\cite{knight2013selfish}, this is not 
+necessarily the case. 
+Players might not necessarily aim to maximise the so called `market share'.
+Rather, the overall service needs to be considered as players aim to minimise
+the overall congestion.
```
Comment (drvinceknight) [inline sections/introduction/main.tex]:
```suggestion
their experienced congestion.
```

File: sections/conclusion/main.tex [hunk 30]
```diff
@@ -1 +1,57 @@
...
+
+The motivation behind this study has been that emergency departments are 
+constantly under a lot of pressure to satisfy some regulations. 
+This paper shows how this can negatively impact the pathway of both the 
+ambulance patients and the ambulance service itself.
+Due to some managerial decision making that takes place at the ED, ambulances 
+stay blocked outside of the ED at the hospital's parking space in an attempt
+to satisfy these regulations.
+
+This study explores a 3-player game theoretic model between the decision makers
```
Comment (drvinceknight) [inline sections/conclusion/main.tex]:
```suggestion
This study explores a generic 3-player game theoretic model between the decision makers 
```

File: sections/conclusion/main.tex [hunk 31]
```diff
@@ -1 +1,57 @@
...
+
+This study explores a 3-player game theoretic model between the decision makers 
+of two queueing systems and a service that distributes individuals to these two
+systems (section \ref{sec:model_overview}).
+It also describes the construction of the underlying queueing theoretic model 
+that has a tandem buffer and a single service centre (section 
+\ref{sec:queueing_model}).
+Furthermore, the formulas for the performance measures of the queueing model 
+are also derived (sections \ref{sec:waiting_time}, \ref{sec:blocking_time}, 
+\ref{sec:proportion_within_target}).
```
Comment (drvinceknight) [inline sections/conclusion/main.tex:19]:
```suggestion
\ref{sec:proportion_within_target}). This novel queuing model is the first contribution of the paper.
```

File: sections/conclusion/main.tex [hunk 32]
```diff
@@ -1 +1,57 @@
...
+The main takeaways from this paper that were observed when playing the game
+between two EDs and the EMS (section \ref{sec:application}) are:
+\begin{itemize}
+    \item Inefficiencies can be learned and emerge naturally
+    \item Targeted incentivisation of behaviours can help escape inefficiencies
+\end{itemize}
+The former relates to the results of asymmetric replicator dynamics that showed 
+that inefficient scenarios can arise by playing the game while the latter 
+implies that the learned inefficiencies can be escaped by carefully applying 
+certain incentives to the players.
```
Comment (drvinceknight) [inline sections/conclusion/main.tex:35]:
```suggestion
certain incentives to the players. 
This is applied game theoretic model is the second main contribution of this paper.
```

File: sections/conclusion/main.tex [hunk 33]
```diff
@@ -1 +1,57 @@
...
+
+This work also has certain limitations that need to addressed.
```
Comment (drvinceknight) [inline sections/conclusion/main.tex]:
```suggestion
```

File: sections/conclusion/main.tex [hunk 34]
```diff
@@ -1 +1,57 @@
...
+The model presented here only assumes the presence of two players that can
```
Comment (drvinceknight) [inline sections/conclusion/main.tex]:
```suggestion
The model presented here assumes the presence of two players that can
```

**Review by drvinceknight — CHANGES_REQUESTED** [review]
This looks good @11michalis11 :+1:

**Pull request merged**